scripts/test.sh

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env bash
 set -e
+set -x
 cd "$(dirname "$0")"
```

Comment:
are changes in this file really vital for this PR?

Reply:
not at all, I just found the test script output a bit unreadable -- I had some errors, but didn't know from which command or test file they originated

Reply:
Can we just revert those? I use `-x` only for debugging purposes, I don't leave that in prod code (and this is how all other scripts are written).